_Week8/Group3/README.md

```diff
@@ -0,0 +1,74 @@
+# WEEK 8 HW - Group 3
+
+## Motivation
+
+For this week's simulation assignment, we chose to illustrate the importance of intracluster correlation and the resulting design effect in clustered research designs.
+
+### ICC
```

Comment:
Great work explaining intracluster correlation. As someone who is not too familiar with the concept and the math behind it, the README does a great job of explaining the concept and walking the reader through your logic, the set up, and how to interpret the results!